free-2.md

```diff
@@ -1,44 +1,77 @@
 # Free project 2
 
-
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+Recipes
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Right now, recipes can range from overly informative ("use a 3.9 inch pan and 
+fry for precisely 2 minutes and 7.3 seconds") to not nearly informative enough
+("fry the food").  This can cause difficulty for the amateur chef - the devil is
+in the details and the details aren't there.  There experience could be improved
+and streamlined if there was a more formal, universal system of specifying how 
+to cook things.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+I would argue that recipes, in general, are alreay a DSL.  They even do a pretty
+good job when they're well written.  My DSL would be a stricter subset of 
+existing recipes designed to create a deterministic (or as close to as possible)
+method of cooking food - ideally a computer could follow these steps exactly and
+produce the same food as a human.
```

Comment:
That would be super cool.  I think the goal of my language, though, was to generate specific enough instructions from user input that a computer could do it.  It would depend a lot on the user's ability to write a clear and complete recipe.